react/features/app/components/AbstractApp.js

```diff
@@ -147,10 +147,12 @@ export class AbstractApp extends Component {
             // Profile is the new React compatible settings.
             const profile = getProfile(this._getStore().getState());
 
-            Object.assign(localParticipant, {
-                email: profile.email,
-                name: profile.displayName
-            });
+            if (profile) {
+                localParticipant.email
```

Comment:
Probably this is not related to the fix but why do we need to have APP.settings functionality and in the same time getProfile?

Is getProfile only work for mobile?

Reply:
It does, but currently only mobile has the app-settings feature that persists these data in localStorage. Eventually it would be great to use the same persistency layer for mobile and web, it just needs some refactor.